### PR TITLE
🛠️ Fix ➾ Clean all docker containers before running any tests

### PR DESCRIPTION
**Pull request by jchenche**
# 🌮 Taqueria PR

Some tests are failing most of the time on Mac Minis but not Linuxes

### CODE REVIEW

**Review by AlirezaHaghshenas — APPROVED** [review]

File: bin/before-all-tests.sh [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 set -e # exiting on error
 
-docker ps # making sure docker is installed, running, and the user has permission
+docker ps > /dev/null # making sure docker is installed, running, and the user has permission
```
Comment (michaelkernaghan) [inline bin/before-all-tests.sh:4]: if docker ps goes to dev null here then the comment should be changed

File: bin/build-all.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ if [ "$0" == "./bin/build-all.sh" ] && [ -f index.ts ]; then
     echo ""
     echo '**********************************************'
     echo "** Checking Dependencies"
-    docker ps
+    docker ps > /dev/null
```
Comment (michaelkernaghan) [inline bin/build-all.sh:12]: if comments on the other file while not in this file too?

**Review by michaelkernaghan — COMMENTED** [review]
two small comments about comments. otherwise fine :)

File: bin/before-all-tests.sh (hunk 1 above)
Comment (jchenche) [reply bin/before-all-tests.sh:4]: @michaelkernaghan changed to what? 🤔 

@AlirezaHaghshenas ideas?

Comment (michaelkernaghan) [reply bin/before-all-tests.sh:4]: making sure docker is not running

File: bin/build-all.sh (hunk 2 above)
Comment (michaelkernaghan) [reply bin/build-all.sh:12]: ok cool

File: bin/before-all-tests.sh (hunk 1 above)
Comment (jchenche) [reply bin/before-all-tests.sh:4]: but that's what the comment is saying, and more. It's not inaccurate

Comment (jchenche) [reply bin/before-all-tests.sh:4]: you mean to say "making sure docker is running"?

**Review by michaelkernaghan — APPROVED** [review]

**Pull request merged**